### DISCUSSION TITLE
Error when: Upload proof of your academic status. Always have same error relate to update billing information though i have already updated

### DISCUSSION THREAD

**Discussion by hongnhu2005** [community/community#125768] [GitHub Education]
### Hi everyone,

<img width="794" alt="Screenshot 2024-05-26 at 11 40 47" src="https://github.com/community/community/assets/170818596/221aeb73-e485-4b09-82b3-07518290cf99">

When i verify github education account. I alaways have same error when upload image proof. I have already updated my github information, i have two factor authentication, billing information, fullname in github profile, but still get same errors

**Comment by y0geshx** [comment]
- Use browser camera to take pictures of your ID..
- Don't upload it for gallary app.
- ID picture should be less than 2mb

I hope this helps you 🙂
